docs/SETUP.md

```diff
@@ -56,10 +56,10 @@ Get your Mapbox access token from Mapbox Dashboard. Put the key in `.env.local`
 To preview the app use:
 
 ```
-yarn serve
+./.veda/veda serve
```

Comment:
I thought they are the same thing! Am I misunderstanding?

Reply:
I was having some trouble with the hot reload capability when I was running `yarn serve` - after saving a file it would not properly reload the browser preview. Then I found the `./.veda/veda` script which appears to add some modifications to all of the basic `yarn` commands including `serve`. After I started using `./.veda/veda serve` the dev experience was much better!